app.js

```diff
@@ -143,14 +155,35 @@ const openModal = ({
 } <i class="modIcon fab fa-pinterest-p hero-fab"></i></a>
     </div>
   </div>
-</div>`;
-  modalDisplay.innerHTML = display;
-};
 
-openMenu.addEventListener('click', show);
+</div>`
+      modalDisplay.innerHTML = display
+    }
+
+openMenu.addEventListener('click', show)
 
-closeMenu.addEventListener('click', close);
-closeModalButton.addEventListener('click', closeModal);
+closeMenu.addEventListener('click', close)
+closeModalButton.addEventListener('click', closeModal)
 openModalButton.forEach((item, index) => {
-  item.addEventListener('click', openModal(arrayData[index]));
-});
+  item.addEventListener('click', openModal(arrayData[index]))
+})
+
+function contactValidation (e) {
+  const email = document.getElementById('email').value
+
+  if (email === '') {
+    e.preventDefault()
+
+    small.innerHTML = 'Hey Comrade! , e-mail field cannot be emty!'
+  } else if (email !== email.toLowerCase) {
+    e.preventDefault()
+
+    small.innerHTML =
+      'Hey Comrade!, e-mail must be small letter, FORM NOT submitted!'
+  } else {
+    small.innerHTML = 'FORM submitted!'
+    small.style.color = '#00ff00'
+  }
+}
```

Comment:
- [ ] [REQUIRED] You have done a great job implementing the validation, however, kindly note that one of the requirements states that: Whenever a user submits the contact form with the email field containing only lowercase characters the form is sent normally. Presently, the form doesn't submit. Kindly ensure the form submits whenever a user inputs an email with lowercase. The submit button/input `click` event is used to attach the function checking the email format. This is good practice and will also enable you to meet up the requirement. 
- [ ] [OPTIONAL] Let's make the error message more professional as your portfolio would be viewed by a lot of professionals. e.g: "email must be lowercase" or "lowercase is required".